package.json

```diff
@@ -20,8 +20,8 @@
     "@types/node": "^6.0.41",
     "fs-extra": "^2.1.2",
     "marked": "^0.3.6",
-    "polymer-analyzer": "=2.0.0-alpha.35",
-    "polymer-editor-service": "=1.4.0"
+    "polymer-analyzer": "^2",
```

Comment:
Why not `^2.0.0` such that format is the same as the other dependencies in this file?